app/models/container_group.rb

```diff
@@ -85,8 +85,6 @@ def disconnect_inv
     _log.info "Disconnecting Pod [#{name}] id [#{id}] from EMS [#{ext_management_system.name}]" \
     "id [#{ext_management_system.id}] "
     self.container_definitions.each(&:disconnect_inv)
-    self.old_ems_id = ems_id
-    self.ext_management_system = nil
```

Comment:
A few lines below, it still does
```
self.old_container_project_id = self.container_project_id
self.container_project_id = nil
```
I suppose for consistency we'll also want to stop that and retain the normal relation, just with one or both sides archived.  And `project.container_groups` association would default to `-> { active }` to preserve current behavior.
@zakiva @zeari what do you think?  Is this the desired end state?